src/java/org/apache/cassandra/io/sstable/CQLSSTableWriter.java

```diff
@@ -614,6 +655,19 @@ public Builder withBuildIndexes(boolean buildIndexes)
             return this;
         }
 
+        /**
+         * Whether the underlying schema which was created for the purpose of the generation should be
+         * removed when the writing is finished. Defaults to true and should not be changed under normal circumstances.
+         *
+         * @param removeSchemaAfterDone true if underlying schema will be removed, false otherwise.
+         * @return this builder
+         */
+        public Builder removeSchemaAfterDone(boolean removeSchemaAfterDone)
```

Comment:
we could have this package protected, it is used from tests only anyway

Reply:
Remember that the builder is a public interface for consumers outside of the C* code tree... this should stay public even if it's only used by tests in C*'s repository.